### DISCUSSION TITLE
📚 Share your testimonials

### DISCUSSION THREAD

**Discussion by Panquesito7** [Panquesito7/Panquesito7#3] [General]
**Welcome!** 👋 

Thanks, everyone for your testimonials, they mean a lot to me. ❤️

**Comment by ayaankhan98** [comment]
I appericiate his work, He is a hard-working individual, who completes his work with great enthusiam and dedication with appericiable programming skills. He is helpful to others and provides response almost instantly whenever required.

**Reply by Panquesito7** [reply]
Thanks so much for the kind words! 😁

**Comment by shuklaritvik06** [comment]
David is a part of many open source communities, where he is constantly contributing and helping other folks whenever they need help. There is so much to learn from him, he has a great enthusiasm for Open Source.

**Reply by Panquesito7** [reply]
Thanks so much for the kind words, Ritvik! ❤️

**Comment by mkubdev** [comment]
David is a person with an incredible consistency of work. In addition to managing an impressive number of communities, he always takes the time to help and improve everything.

**Reply by Panquesito7** [reply]
Thanks so much, Maxime! Your support is what motivates me to keep doing this! ❤️

**Comment by Sonichigo** [comment]
David is a hard-working individual, who completes his work with great enthusiasm and dedication with appreciable programming skills. He is very helpful and also a part of many open source communities, where he is constantly contributing and helping other folks whenever they need help, there is so much to learn from him. I appreciate his work very much.

**Reply by Panquesito7** [reply]
Thanks so much for the support and kind words! ❤️

**Comment by aminoxix** [comment]
David surrounds himself with great people. It is a tribute to his technical and friendly-contributional behaviour that he is able to engage and manage folks around him without misleading or misguidance. He listens to diverging ideas and gathers people to work with the community. :slightly_smiling_face: 

Let's support good changes among communities in future; looking forward to seeing your growth in tech! :computer:

**Reply by Panquesito7** [reply]
Thank you sooo much for the kind words, aminos! ❤️ It's been a great time working with you!

**Comment by YashDaga17** [comment]
David is really helpful.😌
He is very nice and friendly. 💕
He works very hard with his projects to make it successful.
Glad to know a person like David 🤗

**Reply by Panquesito7** [reply]
Thank you for the very nice words! ❤️

**Comment by aparna2071** [comment]
David is really helpful who helps folks in their open source journey by suggesting changes to their PR in a very friendly way.
He has also approved some of my PRs.
His contributions to various communities are amazing, much to learn from him.

**Reply by Panquesito7** [reply]
Thanks so much for the kind words! ❤️
You also do great work in communities!

**Comment by NS007-dev** [comment]
David has been there since I began my journey on GitHub, reviewing and supporting my code, and guiding me in the right way in open source. There has never been a pull request I have submitted, without him placing a review and helping me, or approving. 
Thank you David for all your support, and I look forward to collaborating more with you.

**Reply by Panquesito7** [reply]
Thank you so much for the very kind words! ❤️

**Comment by mahinperwaiz** [comment]
DAVID is one of the people who encouraged me to get into open-source and GitHub in general. A true legend in my opinion. Being a community manager of 30+ communities is just another level. Having 5000+ contributions on GitHub is no joke. I am really looking forward to get mentored by him and do my first PR.

**Reply by Panquesito7** [reply]
Thanks so much for the very amazing words! I'm glad you got into the open-source world! ❤️ 🎉

**Comment by E-fais** [comment]
David is  friendly mentor for me , he doesn’t even know me , but he i has helped me to learn open source.

**Reply by Panquesito7** [reply]
Thank you for the kind words! ❤️ Any assistance or help you need, let me know!

**Comment by suraj-996** [comment]
I came to your profile only today and was amazed to see how hard-working you are and you are doing great in your life. I loved connecting with you and have also started contributing to open source.

**Reply by Panquesito7** [reply]
Thanks so much for the amazing words! ❤️ Glad to have you in the open-source world.
Any help or assistance you need, you can contact me via Discord and we can have a talk! Panquesito7#3723

**Comment by Panquesito7** [comment]
You can now submit a testimonial both here or at my [LinkFree](https://github.com/EddieHubCommunity/LinkFree/issues/new?labels=testimonial&template=testimonial.yml&title=New+Testimonial+for+David%20Leal&name=Panquesito7) profile! 🎉
This will help me grow my LinkFree profile and have your testimonials shown to other people.

Thank you for your support! ❤️

**Comment by thuongtruong1009** [comment]
Oh, by chance from searching on Github, I came across your profile by accident. It's quite special. It can be seen that you are a regular contributor to the open source community, very enthusiastic... I hope to cooperate with you to develop and contribute more in the future.

**Reply by Panquesito7** [reply]
Wow! 👀 Didn't really expect my profile to be found like that, hehe.
Still, thank you for the amazing testimonial and words! 😁 

Feel free to talk via Discord, and we can collaborate on projects! 🚀

**Comment by prituyadav** [comment]
David, aka Panquesito7, is a true gem in the open-source community. With his exceptional technical skills and friendly demeanor, he effortlessly brings people together, listens to diverse perspectives, and leads by example. His dedication to fostering positive change is commendable, and I am excited to witness his remarkable growth in the tech industry. 💻

**Reply by Panquesito7** [reply]
This made my day! ❤️ Thank you so much for the amazing words! It motivates me a lot! 😄
